css/index.css

```diff
@@ -133,4 +133,65 @@ button:hover{
 }
 .artist img{
```

Comment:
Purely bc I saw how attentive you were to details, your images are as wide as the text below them. The design sheet showed the text being wider than the image. I'm thinking that making this a smaller width of it's parent *might* fix it, but  not entirely sure. Personally though, I prefer the look of how you did it tbh.